Backport new release notes structure to develop (#1316)
Avaiga/taipy-doc: docs/release-notes/4.1/index.md

---
title : Release Notes for version 4.1
---

> This is the list of changes to Taipy version 4.1.

Reviewer: ```suggestion
These are the updates and changes introduced in Taipy version 4.1.
```
